src/script/components/avatar/UserAvatar.test.tsx

```diff
@@ -91,7 +91,7 @@ describe('UserAvatar', () => {
   });
 
   it('shows avatar badge for connection request', async () => {
-    const participant = new User('id', null);
+    const participant = new User('id', undefined);
```

Comment:
you could probably do `new User('id')` ;) (also valid for all the other ts fixes)

Reply:
u're right, no need to pass undefined explicitly 👍 resolved